plugin/web/extensions/sidebar/modals/datacatalog/components/index.tsx

```diff
@@ -54,13 +73,33 @@ const DataCatalog: React.FC = () => {
           expandedFolders={expandedFolders}
           searchTerm={searchTerm}
           filter={filter}
+          dataSource={"plateau"}
+          editable={!isCustomProject}
```

Comment:
Doesn't this make it so plateau is not editable if there is a city (custom) project? I don't think those things are connected.
If the user has inputted all fields for plateau (including a token) this should be editable. Doesn't matter if custom dataset info is also added or not.